WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerFilterCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+
+class LayoutPickerFilterCollectionViewCell: UICollectionViewCell {
+
+    static var cellReuseIdentifier: String {
+         return "LayoutPickerFilterCollectionViewCell"
+     }
+
+    static var nib: UINib {
+        return UINib(nibName: "LayoutPickerFilterCollectionViewCell", bundle: Bundle.main)
+    }
+
+    static var font: UIFont {
+        return WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .semibold)
+    }
+
+    static func estimatedWidth(forFilter filter: GutenbergLayoutSection) -> CGFloat {
+        /// The emoji below is used as a placeholder to estimate the size of the title. We don't use the actual emoji provided by the API because this could be nil
+        /// and we want to allow space for a checkmark when the cell is selected.
+        let size = "👋 \(filter.section.title)".size(withAttributes: [
+            NSAttributedString.Key.font: font
+        ])
+
+        return size.width + 32
```

Comment:
If the hard-coded value could be turned into a named constant I think that would make this easier to maintain. It's hard to hell of the 32 is used to provide 16pts of margin either side, or something else.